Xamarin.Forms.Controls/Xamarin.Forms.Controls.csproj

```diff
@@ -22,6 +22,10 @@
   </PropertyGroup>
   <ItemGroup>
     <None Remove="BuildNumber.txt" />
+    <None Remove="Fonts\CuteFont-Regular.ttf" />
+    <None Remove="Fonts\Dokdo-Regular.ttf" />
+    <None Remove="Fonts\fa-regular-400.ttf" />
+    <None Remove="Fonts\PTM55FT.ttf" />
```

Comment:
I think we can remove this entire ItemGroup

Reply:
VS4mac added that. I tried removing it. Then they were duplicated.

Reply:
Yea I'm trying it on vsmac as well and it's not reacting so well :-/ 

So we can leave for now